packages/dds/cell/src/cell.ts

```diff
@@ -229,12 +307,20 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
                 this.pendingMessageIds.shift();
                 // We got an ACK. Update messageIdObserved.
                 this.messageIdObserved = cellOpMetadata.pendingMessageId;
+                // update the attributor
+                if ((this.options?.attribution?.track) ?? false) {
```

Comment:
seems reasonable to move this check to inside `this.setAttribution`. saves a couple bytes of code.